integration/port_forward_test.go

```diff
@@ -55,10 +55,28 @@ func TestPortForward(t *testing.T) {
 	portforward.WhiteBoxPortForwardCycle(t, kubectlCLI, ns.Name)
 }
 
-// TestPortForwardDeletePod tests that port forwarding works
+func TestRunPortForwardDeletePod(t *testing.T) {
+	if testing.Short() || RunOnGCP() {
+		t.Skip("skipping kind integration test")
+	}
+
+	ns, _, deleteNs := SetupNamespace(t)
+	defer deleteNs()
+
+	stop := skaffold.Run("--port-forward").InDir("examples/microservices").InNs(ns.Name).RunBackground(t)
```

Comment:
you'll need to add some more flags to enable the API server: e.g. `--rpc-http-port 50051 --enable-rpc=true` - but I'd rather you use a random port like in the other test.